src/angular/projects/spark-extras-angular-dictionary/package.json

```diff
@@ -19,7 +19,7 @@
     "@angular/platform-browser": "^6.1.4",
     "@angular/platform-browser-dynamic": "^6.1.4",
     "@angular/router": "^6.1.4",
-    "@sparkdesignsystem/spark-core-angular": "^7.0.0",
-    "@sparkdesignsystem/spark-dictionary": "^1.0.0"
+    "@sparkdesignsystem/spark-core-angular": "^8.0.0",
+    "@sparkdesignsystem/spark-dictionary": "^1.0.1"
```

Comment:
Dictionary is at 1.0.1 and this was outdated. https://www.npmjs.com/package/@sparkdesignsystem/spark-dictionary